### ISSUE TITLE
vim keybindings

### ISSUE THREAD

**Issue by acarl005** [asciimoo/wuzz#22]
This is awesome! I hate to be "that guy" but vim keybindings would be nice 🤓. Vim's `<ctrl-w>` window commands for navigating windows would be especially convenient for quickly jumping around.

**Comment by Benaiah** [comment]
I have an existing branch with j/k keybindings for scrolling up and down - adding a `<ctrl-w>` keybind would be pretty easy. I messed up some stuff in git, but I'll try to get that finished up and send in a pull request today.

**Comment by Benaiah** [comment]
@acarl005 did you have any other suggestions for vim keybinds (other than in the text fields themselves, not sure how feasible that would be)?

**Comment by acarl005** [comment]
Apart from the window commands, the ability to navigate (`hjkl` and `bweBWE`) within the text fields would be the most useful imho. It would be nice to be able to quickly copy-paste text blocks, e.g. copying a whole JSON object with `ya{` to the system clipboard if possible.

**Comment by asciimoo** [comment]
Maybe, it would be better to make keybindings configurable in the future.
I love vim too, but unfortunately not everybody feels the same, and they may not find it intuitive.

**Comment by acarl005** [comment]
You're absolutely right. I'd be happy with a command line option like `wuzz --vim-keys` or something like that and I could just alias it.

**Comment by Benaiah** [comment]
@asciimoo I agree with making keybinds configurable. wuzz could use a config file for quite a few things - for instance, settings like pretty-printing seem like they would be a bit awkward to provide on the command line.

**Comment by caedn** [comment]
I see that ctrl-j/k jump to the next/previous field, I think ctrl-h/l to jump between columns would be very useful as well.

**Comment by justinas** [comment]
Alternatively, I would suggest an option of Readline/Emacs keybindings. Ctrl-A, Ctrl-E, Alt-F, Alt-B like in the shell would be super useful.

**Comment by nigurr** [comment]
Can we add support for vim? so that delete line is easy :) currently i don't have option for this

**Comment by ieatmumble** [comment]
Hi,

Indeed, it would be really helpful to be able to delete one row of text, along the lines of `dd` in vim or `ctrl`+`k` in nano, at least for the cases when the text is pasted from a different source, in bulk.
Along the same lines, being able to delete everything to the left on the same row and everything to the right would be extra helpful.

Thanks a lot for this useful and promising tool!
